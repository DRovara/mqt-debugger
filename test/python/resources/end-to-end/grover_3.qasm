qreg q[2];
qreg flag[1];
creg c[2];

// initialization
h q;
x flag;
barrier q;

// oracle: mark target state |11>
h flag;
ccx q[0], q[1], flag;
h flag;
barrier q;

// diffusion
h q;
x q;
h q[1];
cx q[0], q[1];
h q[1];
x q;
h q;

measure q -> c;
