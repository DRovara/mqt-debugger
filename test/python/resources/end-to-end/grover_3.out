0,0,0,0,0,0,0,-1
---
c[0] = 1,c[1] = 1
---
0
